File: atcoder_python/C_Multiple_Gift.py
#           C - Multiple Gift
# ----------------------------------------
# 問題
# https://atcoder.jp/contests/abc083/tasks/arc088_a

# シンプルに実装できた
# AC
# ----------------------------------------

# input
X, Y = map(int, input().split())

# solve
counter = 0
while X <= Y:
    X *= 2
    counter += 1

print(counter)

File: arihon/chapter2/section2/C_Multiple_Gift.cpp
//            C - Multiple Gift            
// ----------------------------------------
// 問題
// https://atcoder.jp/contests/abc083/tasks/arc088_a
// ----------------------------------------

#include <bits/stdc++.h>
using namespace std;
#define rep(i, n) for (int i = 0; i < (int)(n); i++)
#define ALL(A) A.begin(), A.end()
typedef long long ll;
template <typename T> inline bool chmax(T &a, const T b) { if (a < b) { a = b; return true; } return false; }
template <typename T> inline bool chmin(T &a, const T b) { if (a > b) { a = b; return true; } return false; }
constexpr int MOD = 1000000007;
constexpr int mod = 998244353;

int main() {
    ll X, Y; cin >> X >> Y;
    int ans = 0;
    while (X <= Y) {
        X <<= 1;
        ans++;
    }

    cout << ans << endl;
}
